extensions/amp-story/1.0/amp-story.js

```diff
@@ -1846,6 +1846,9 @@ export class AmpStory extends AMP.BaseElement {
         if (!this.backgroundBlur_) {
           this.backgroundBlur_ = new BackgroundBlur(this.win, this.element);
           this.backgroundBlur_.attach();
+          if (this.activePage_) {
+            this.backgroundBlur_.update(this.activePage_.element);
```

Comment:
If a media from this page is already used for the blur, will it early return or animate again?

Reply:
We remove the blur from the dom when it's not the supported UI type:
https://github.com/ampproject/amphtml/blob/main/extensions/amp-story/1.0/amp-story.js#L1824-L1825

It will draw the blur again but will not animate since it's the first update.